pandemania/src/FirebaseConfig.js

```diff
@@ -0,0 +1,10 @@
+const config = {
```

Comment:
firebase config file will be added with the firebase feature, please remove it from this commit